### ISSUE TITLE
network:online not fired

### ISSUE THREAD

**Issue by paulbonneau** [uncatcrea/wp-appkit#306]
The network:online event is not fired on both my desktop and mobile app with this code.
```
    App.on('network:online', function(event){
        alert("connected to the net");
        // Get the current network state
        var ns = TemplateTags.getNetworkState(true);
        // Display the current network state
        showMessage(ns);
    });
```
I tried with and without adding the corresponding cordova plugin in my app conf but it didn't change nothing.

`<gap:plugin name="cordova-plugin-network-information" source="npm" />`

**Comment by blupu** [comment]
Hi,

- Your code is correct
- Just retested it and it worked on my side (both preview in Chrome and on phones)
- Not using the Cordova plugin should default to the browser support (generic yes/no rather than giving the type of connection)

So, I'll need more info to solve your issue (things like WP-AppKit version, is this function in functions.js, what theme do you use, what platform are you targeting, etc.).

**Comment by paulbonneau** [comment]
- I use Version 1.0.2.
- The theme is the q-android theme.
- I want to display this notification on desktop (in order to test the feature) and on android devices.

**Comment by blupu** [comment]
ok - that's what I use too.

- Do you preview in Chrome with emulation mode activated?
- Is there a way I can access your preview online?

**Comment by paulbonneau** [comment]
Yes I preview in chrome with the emuation mode (but it doesn't work with or without this mode).

Unfortunately, I can't give you the access to the online preview. I'll try to replicate the error on a personal project and send you a link tonight. Thanks for your answers !

**Comment by blupu** [comment]
ok - your issues are weird but we'll work to solve them. A preview will help for sure.
